### ISSUE TITLE
422 Unprocessable Entity error when trying to launch catalog entry

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#6818]
**Rancher Version:**
v1.2.0-rc1
**Docker Version:**
1.12
**OS and where are the hosts located? (cloud, bare metal, etc):**
ubuntu 16.04
**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
single node rancher
**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**
Cattle
**Steps to Reproduce:**
- add https://github.com/OpenSaasAU/community-catalog.git catalog
- try to launch openedx catalog entry with default configuration

**Results:** 
You will get an error:
```
 Error (Bad response statusCode [422]. Status [422 Unprocessable Entity]. Body: [code=MissingRequired, fieldName=lbConfig, baseType=error] from [http://localhost:8080/v2-beta/projects/1a5/loadbalancerservices])
```
Running the catalog entry on v1.1.4 didn't create the same problem.

**Comment by deniseschannon** [comment]
Fixed in rancher-compose v0.11.0-rc15

**Comment by deniseschannon** [comment]
Tested same template with v0.11.0-rc15. The error doesn't appear.
